src/components/signup/SignupState.js

```diff
@@ -602,7 +606,7 @@ const Signup = ({ navigation }: { navigation: any, screenProps: any }) => {
         return navigateWithFocus(nextRoute.key)
       } catch (e) {
         log.error('email verification failed unexpected:', e.message, e, { dialogShown: true })
```

Comment:
add decorate

Reply:
It already contains an understandable error message.